src/main/java/com/team701/buddymatcher/controllers/communication/CommunicationController.java

```diff
@@ -89,6 +89,18 @@ public CommunicationController(CommunicationService communicationService, UserSe
         }
     }
 
+
+    @Operation(summary = "Delete method to delete a user's conversation")
+    @DeleteMapping(path = "/messages/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Void> deleteUserMessages(@Parameter(hidden = true) @SessionAttribute("UserId") Long userId, @PathVariable("id") Long buddyId) {
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteUserMessages(@Parameter(hidden = true) 
                                                   @SessionAttribute("UserId") Long userId, 
                                                   @PathVariable("id") Long buddyId) {
```
Line 95 is against the code convention of this repository. It extends beyond the 120 char limit. The suggested change hardwraps line 95 to not extend beyond the 120 char line limit.